pandas/_libs/src/khash.pxd

```diff
@@ -11,13 +11,13 @@ cdef extern from "khash_python.h":
         PyObject **keys
         size_t *vals
 
-    inline kh_pymap_t* kh_init_pymap()
-    inline void kh_destroy_pymap(kh_pymap_t*)
-    inline void kh_clear_pymap(kh_pymap_t*)
-    inline khint_t kh_get_pymap(kh_pymap_t*, PyObject*)
-    inline void kh_resize_pymap(kh_pymap_t*, khint_t)
-    inline khint_t kh_put_pymap(kh_pymap_t*, PyObject*, int*)
-    inline void kh_del_pymap(kh_pymap_t*, khint_t)
+    kh_pymap_t* kh_init_pymap()
+    void kh_destroy_pymap(kh_pymap_t*)
```

Comment:
But they still trigger the warning when building. So apparently that does not matter